README.md

```diff
@@ -48,7 +48,7 @@ Use the batch execution strategy with your schema
 ```ruby
 MySchema = GraphQL::Schema.new(query: MyQueryType)
 MySchema.query_execution_strategy = GraphQL::Batch::ExecutionStrategy
```

Comment:
Or I guess not, since we're still dispatching into it for mutation sub-query resolution.